sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -60,91 +51,91 @@
       <dependency>
         <groupId>com.azure.spring</groupId>
         <artifactId>azure-spring-boot</artifactId>
-        <version>${azure.spring.boot.integration.version}</version>
+        <version>3.4.0-beta.1</version> <!-- {x-version-update;com.azure.spring:azure-spring-boot;current} -->
```

Comment:
Can tag work if we use eng/versioning/set_version.py to update version?

Reply:
It can work.